src/pages/Swap.vue

```diff
@@ -446,11 +464,15 @@ export default defineComponent({
             const type = transactionReceipt.status === 1
                 ? 'success'
                 : 'error';
-            const link = getEtherscanLink(transactionReceipt.transactionHash);
-            store.dispatch('ui/notify', {
-                text,
-                type,
-                link,
+            
+            store.dispatch('ui/updateNotification', {                
+                id: notifyId,
+                notification: {
+                    text,
+                    type,
+                    link,
+                },
+                removeTopNotification: true,
```

Comment:
with the `notifyId` we got from the notification, we can make an update to it (changing its status from `pending` -> `success`)